exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/transformers/model_management.ipynb

```diff
@@ -0,0 +1,225 @@
+{
```

Comment:
<p>Now we have at least <strong>one</strong> model installed ... (... you probably will have more <strong>models</strong> installed).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/418/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Done</p>